ergo/distributions/histogram.py

```diff
@@ -182,14 +233,29 @@ def to_lists(self, true_scale=True, verbose=False):
 
         return xs, ps
 
-    def to_pairs(
-        self, true_scale=True, verbose=False,
-    ):
+    def to_pairs(self, true_scale=True, verbose=False, total_p: float = 1.0):
```

Comment:
Doesn't feel right to add this transformation here. Options that seem better:

1. Make a separate function that rescales the pairs
2. Support histograms that don't sum to 1 more generally, add a rescale method to HistogramDist that returns a rescaled histogram, then just export using the old `to_pairs` function

Reply:
1 seems good, I think 2 seems overengineered